Consolidate mac_dot into dot

atdillinger opened the pull request:
The majority of functionality and installs are the same between the hardware. This should make it easier to keep the systems in-check.

atdillinger committed:
Consolidate mac_dot into dot

The majority of functionality and installs are the same between the
hardware. This should make it easier to keep the systems in-check.

dillingerlab commented:
Archiving [mac_dot](https://github.com/dillingerlab/mac_dot/tree/main) after merge

atdillinger committed:
git improvements

add rerere
fix autocorrect

atdillinger committed:
Document Local Neovim Configs

There must be a better way but this works for now.

atdillinger committed:
Recipe for LazyVim setup

Create recipe for lazyvim config. I want to remap the keys but telecope
and other plugins do not play well with the simple answer... Once I'm
more familiar I'll try again

atdillinger reviewed (commented):
Manual validation FTW...
